ironfish/src/rpc/clients/ipcClient.ts

```diff
@@ -23,11 +23,12 @@ export class RpcIpcClient extends RpcSocketClient {
   onError = new Event<[error: unknown]>()
 
   constructor(
+    rpcAuthToken: string,
```

Comment:
ipc client doesn't use auth so we should remove this

Reply:
removed